vars/main.yaml

```diff
@@ -0,0 +1,66 @@
+---
+php_fpm_pools:
+- name: fpm-host
+  listen: /run/php/php7.0-fpm.sock
+  user: www-data
+  pm: static
+  pm_max_children: 20
+  pm_start_servers: 20
+  processes_priority: -19 
+php_fpm_php_ini_values:
+- section: PHP
+  option: extension
+  value: php_mysqli
+- section: PHP
+  option: extension
+  value: php_mbstring
+- section: PHP
+  option: extension
+  value: php_dblib
+- section: PHP
+  option: extension
+  value: php_ldap
+- section: PHP
+  option: extension
+  value: php_gd
+- section: PHP
+  option: extension
+  value: php_imap
+- section: PHP
+  option: extension
+  value: php_zip
+- section: PHP
+  option: extension
+  value: php_zlib
+nginx_access_log: "/var/log/nginx/limesurvey_access.log"
+nginx_error_log: "/var/log/nginx/limesurvey_error.log"
+served_domains:
```

Comment:
I bet most of this file actually belongs into the hostvars.

Reply:
done